test/compilable/dtoh_ClassDeclaration.d

```diff
@@ -149,6 +149,27 @@ class Child final : public Parent
 public:
     void foo() /* const */;
 };
+
+class VisitorBase
+{
+public:
+    virtual void visit();
+};
+
+class VisitorInter : public VisitorBase
+{
+public:
+    using visit = VisitorBase::visit;
+    virtual void visit(int32_t i);
+};
+
+class Visitor : public VisitorInter
+{
+public:
+    using visit = VisitorInter::visit;
```

Comment:
This doesn't work right now because the function alias seems to skip the intermediate class and hence produces `VisitorBase::visit`

Reply:
This usually looks like this: https://github.com/dlang/dmd/blob/646d83480f27f7a7b48526c9a81ad2a2c83b3826/src/dmd/visitor.h#L577